### ISSUE TITLE
Cannot addColumn with a primaryKey on Postgres

### ISSUE THREAD

**Issue by Nelspike** [db-migrate/node-db-migrate#408]
Hello!

After browsing the issues, I found a similar issue posted a couple of days ago (https://github.com/db-migrate/node-db-migrate/issues/407). However, this one's different, as it has to do with postgres (psql).

It seems that I cannot add a column while referencing it to be a primary key while using `addColumn`. To make matters more precise, the error is:

```
[ERROR] TypeError: Cannot read property 'emitPrimaryKey' of undefined
    at Class.Base.extend.createColumnConstraint (.../node_modules/db-migrate/lib/driver/pg.js:200:39)
    at Class.Base.extend.createColumnDef (.../node_modules/db-migrate/lib/driver/pg.js:29:31)
    at Class.module.exports.Class.extend.addColumn (.../node_modules/db-migrate/lib/driver/base.js:178:20)
    at exports.down (.../migrations/20160829085748-myMigration.js:16:8)
    at Migration._down (.../node_modules/db-migrate/lib/migration.js:176:34)
```

And the trace goes on. However, I debugged it a bit, and found the error to be the non-passing of objects, therefore resulting on the undefined. Step by step:

My migration:

```
db.addColumn('table_name', 'column_name', { type: type.STRING, primaryKey: true }, callback);
```

Where the error occurs: https://github.com/db-migrate/node-db-migrate/blob/bd942bca54fb1d85592e840999529449ac12f055/lib/driver/pg.js#L197

It would seem options are not passed here: https://github.com/db-migrate/node-db-migrate/blob/bd942bca54fb1d85592e840999529449ac12f055/lib/driver/base.js#L178

Therefore resulting on the undefined.

Keep up the good work, and cheers!

**Comment by wzrdtales** [comment]
Can you tell me which db-migrate version you've used?

**Comment by Nelspike** [comment]
@wzrdtales - the one on the link's commit (0.9.25). Seems like it's an old issue though!

**Comment by wzrdtales** [comment]
Ok, I will most probably review and reproduce this tomorrow, did you already tried this on the v0.10.x track?

**Comment by Nelspike** [comment]
Works wonders with the newest versions, aye! Most of the stack I work with uses the 0.9.x ones, though. would it be wise to fix push out a fix within a patch for this for older versions?

**Comment by wzrdtales** [comment]
I will c if I will backport this to the v0.9.x track. As long as 0.10.x is not released on the main track 0.9.x is still supported though.

**Comment by wzrdtales** [comment]
Was backported and released under the latest 0.9.x track.
